modules/cudf/src/series.ts

```diff
@@ -526,11 +530,31 @@ export {
   StructSeries,
 };
 
-function asColumn<T extends DataType>(value: SeriesProps<T>|Column<T>|arrow.Vector<T>): Column<T> {
+function inferType(value: any[]): DataType {
+  if (value.length == 0) return new Float64;
+  if (value.every((val) => typeof val === 'string' || val === null)) return new Utf8String;
+  if (value.every((val) => typeof val === 'number' || val === null)) return new Float64;
+  if (value.every((val) => typeof val === 'bigint')) return new Int64;
```

Comment:
Testing for `x == null` will handle the case where `x` is either `null` or `undefined`. It's probably good to accept both, because someone could do `Series.new(["foo", , "bar"])`, and that middle slot will be `undefined`, not `null`.

```suggestion
  if (value.every((val) => typeof val === 'string' || val == null)) return new Utf8String;
  if (value.every((val) => typeof val === 'number' || val == null)) return new Float64;
  if (value.every((val) => typeof val === 'bigint' || val == null)) return new Int64;
```

Reply:
fixed